codeql.yml

```diff
@@ -0,0 +1,28 @@
+paths:
+  - app
+  - config
+  - lib
+  - components
```

Comment:
do we need to add project_gems? we have some baggage there

Reply:
Good catch, we should look into that.

Reply:
btw: I forgot to add but I don't see it as a blocker, we can merge it as it is and add it later